lib/elixir/lib/enum.ex

```diff
@@ -2259,14 +2266,12 @@ defmodule Enum do
     do: []
   def take_random(first..first, count) when is_integer(count) and count >= 1,
     do: [first]
-  def take_random(first..last, 1) when first > last,
-    do: take_random(last..first, 1)
-  def take_random(first..last, 1),
-    do: [random_index(last - first) + first]
+  def take_random(_.._ = range, 1),
+    do: [random(range)]
```

Comment:
thank you José, but that wouldn't work. take_random returns unique values, and it can never return more element than the original enumerable had.
that would be the implementation for ranges for `Enum.random/2` which is a new function I was planning on propose after this PR. To contrast with `Enum.take_random/2` in the sense that this function may return repeated values, and it will return a list of the exact number of count elements given.